### PR TITLE
GEOPY-1119

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1119 - **

### CODE REVIEW

File: omf/fileio/geoh5.py [hunk 1]
```diff
@@ -647,18 +647,23 @@ def collect_omf_attributes(element: MappedData, **kwargs) -> dict:
...
-            count + 1: str(val) for count, val in enumerate(element.legends[0].values)
-        }
-        color_map = np.vstack(
-            [
-                np.r_[count + 1, val, 1.0]
-                for count, val in enumerate(element.legends[1].values)
-            ]
-        )
+        value_map = {}
+        color_map = [np.r_[0, 0, 0, 0, 0]]
```
Comment (MatthieuCMira) [inline omf/fileio/geoh5.py:651]: Just a question: 
we are appending a new count for every. Why is the first raw full of zeros?
For unknown?
Is unknown always zero? Or could it be something else?
Heard JP already have some bug about unknown being not zeros

File: omf/fileio/geoh5.py [hunk 2]
```diff
@@ -585,7 +585,7 @@ def collect_attributes(
         """
         with fetch_h5_handle(workspace):
             if isinstance(element, UidModel):
-                values = np.r_[getattr(element, "array")]
+                values = element.array.array
```
Comment (MatthieuCMira) [inline omf/fileio/geoh5.py:588]: guess array.array is legit there? a bit strange

**Review by MatthieuCMira — COMMENTED** [review]
Just one question.

You test are not passing.
I'll approve once they will be passed.

File: omf/fileio/geoh5.py (hunk 1 above)
Comment (domfournier) [reply omf/fileio/geoh5.py:651]: That's what this issue is addressing. First value is in the index (reserved for GA 0=="Unknwow". The color doesn't really matter [0, 0, 0] is black, and the alpha is 0 (transparent).

File: omf/fileio/geoh5.py (hunk 2 above)
Comment (domfournier) [reply omf/fileio/geoh5.py:588]: OMF already stores it as a numpy array. That np.r_ conversion was slowing down a ton, going from a non-numpy to np.ndarray. OMF already stores it as a numpy array, so no need to convert.

**Review by MatthieuCMira — APPROVED** [review]
Okay I guess I have no more review.

I heard JP complaining about the rigidity of the unknown category in GA, but that's out of our scope.

Accept

File: omf/fileio/geoh5.py (hunk 1 above)
Comment (MatthieuCMira) [inline omf/fileio/geoh5.py:651]: Question here:

what if unknown somewhere else than index=0.

Could happened if GA change a bit. 

Could be nice to map colormap ased on the index which are always sorted.

**Review by MatthieuCMira — COMMENTED** [review]
A question occurs

File: omf/fileio/geoh5.py (hunk 1 above)
Comment (domfournier) [reply omf/fileio/geoh5.py:651]: The "unknown" is dealt with after if it exists on the omf file, but I see your point. The dict for value_map could be instantiated the same.

**Review by MatthieuCMira — COMMENTED** [review]
Ok for me

**Review by MatthieuCMira — APPROVED** [review]
approve

**Pull request merged**